CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Contributing to USWDS web components
+
+Thank you for your interest in contributing to USWDS web components.
+
+> [!Important]
+>
+> **This repo is currently in beta and is not yet ready for community code contributions.**
+>
+> While the USWDS core team works through big decisions about architecture and style, we ask that you do not submit pull requests. Instead, we encourage you to contribute by your sharing your ideas, concerns, questions, and experiences with the USWDS core team and community.
+>
+> Learn more about how you can provide feedback in the [submitting an issue](#submitting-an-issue) and [starting a discussion](#starting-a-discussion) sections below.
+
+## How can I contribute?
+
+### Submitting an issue
+
+If you have discovered a bug or have a new feature to suggest, let us know by [submitting an issue on GitHub](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue). We’ll address it as soon as we are able (but because we are in beta, please be aware that it might be awhile).
+
+### Starting a discussion
+
+If you want to share ideas, questions, or concerns with the USWDS core team and community, [open a discussion on GitHub](https://docs.github.com/en/discussions/quickstart#creating-a-new-discussion).
+
+### Submitting a pull request
+
+At this time, we are not accepting code contributions in the community. We expect to open up community pull request submissions in early 2025, so check back later for instructions on how to contribute pull requests to USWDS web components.
+
+## Code of Conduct
+
+USWDS is committed to building a safe, welcoming, harassment-free culture for everyone. USWDS is a part of the Technology Transformation Services (TTS) within the General Services Administration (GSA) and we expect everyone to follow the GSA TTS Code of Conduct.
+
+We encourage you to read this project’s Contribution Guide (you are here) and its [README](https://github.com/uswds/uswds-next/blob/develop/README.md). If you want to read more about our open source policy or have questions, check out the [18F Open Source Policy GitHub repository](https://github.com/18f/open-source-policy) or send us an [email](uswds@gsa.gov).
```

Comment:
**note:** This is mostly pulled from [uswds' contributing.md file](https://github.com/uswds/uswds/blob/develop/CONTRIBUTING.md#code-of-conduct).